force-app/main/default/triggers/OpportunityTrigger.trigger

```diff
@@ -0,0 +1,54 @@
+/* Opportunity trigger should do the following:
+* 1. Validate that the amount is greater than 5000.
+* 2. Prevent the deletion of a closed won opportunity for a banking account.
+* 3. Set the primary contact on the opportunity to the contact with the title of CEO.*/
+
+
+trigger OpportunityTrigger on Opportunity (before insert, before update, before delete, after update){
+    If(Trigger.isBefore && Trigger.isUpdate){
+        For(Opportunity opp : Trigger.new){
+            System.debug('For before and update opp is ' + opp);
+            If (opp.amount < 5000){
```

Comment:
For the future, how could we make this value into something an end User can configure or a developer can easily change? We call values like this a "magic number" because we don't know what the number really means beyond how it functions in the context of the code.